src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -139,6 +139,31 @@ public IntervalMergingRule getIntervalMergingRule() {
         return intervalMergingRule;
     }
 
+    /**
+     * Returns the full set of traversal intervals specified on the command line, including parsed intervals without
+     * merging intervals specified by the user on the command line. This is an advanced use case
+     *
+     * NOTE: this currently does not account for interval exclusion arguments
+     *
+     * @param sequenceDict used to validate intervals
+     * @return the full set of intervals specified on the command line, without any merging performed
+     */
+    public List<SimpleInterval> getIntervalsWithoutMerging(final SAMSequenceDictionary sequenceDict ) {
+        if (!intervalsSpecified()) {
+            throw new GATKException("Cannot call getIntervalsWithoutMerging() without specifying either intervals to include or exclude.");
```

Comment:
This should check whether `getIntervalStrings()` is non-empty to make sure that there are intervals to include. The error message needs to be adjusted as well:

"without specifying either intervals to include or exclude" -> " without specifying intervals to include"

Should the method also throw if there are exclusion intervals specified, or should it just silently ignore them? I leave that up to you...